README.md

```diff
@@ -1,11 +1,11 @@
 # WP-Droid [WIP]
 
-A project to convert your Wordpress website into an android app. This app uses [WP API](v2.wp-api.org/) to work properly.
-The app is just an implementation of how wordpress api can be used to extend wordpress api and convert blog into an app.
+A project to convert your wordpress website into an android app. This app uses the [WP API](v2.wp-api.org/) to work properly.
```

Comment:
I guess we should make it uppercase to be in sync with rest of the changes.

Reply:
Although I tried to change everywhere but if you feel so then we can keep it as in Uppercase. Rather than "Wordpress", lets update it to "WordPress".